explore/precision.py

```diff
@@ -356,12 +402,14 @@ def add_function(name, mp_function, np_function, ocl_function,
     ocl_function=make_ocl("return fmod(q, 2*M_PI);", "sas_fmod"),
 )
 add_function(
-    name="debye",
+    name="gauss_coil",
     mp_function=lambda x: 2*(mp.exp(-x**2) + x**2 - 1)/x**4,
     np_function=lambda x: 2*(np.expm1(-x**2) + x**2)/x**4,
     ocl_function=make_ocl("""
     const double qsq = q*q;
-    if (qsq < 1.0) { // Pade approximation
+    // For double: use O(5) Pade with 0.5 cutoff (10 mad + 1 divide)
+    // For single: use O(7) Taylor with 0.8 cutoff (7 mad)
+    if (qsq < 0.0) {
         const double x = qsq;
         if (0) { // 0.36 single
```

Comment:
lots of redundant code - please remove the branches with if(0){...}

Reply:
This is documentation as code, not part of the mainline library.  The code which uses it, sasmodels/models/mono_gauss_coil.c, has the comment: "See explore/precision.py, gauss_coil function" so an interested party can confirm our choice of approximation by checking the different orders.